### PR TITLE
Testing_Bears.rst: Change check_validity docs

### PR DESCRIPTION
**Pull request by stobis**
This changes the description of ``check_validity`` in Example 2
to avoid ambiguity and helps understand its behavior by
short explanations for both usages from example.

Closes https://github.com/coala/coala/issues/5439

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: docs/Developers/Testing_Bears.rst [hunk 1]
```diff
@@ -93,15 +93,17 @@ arguments.
...
-``check_validity`` asserts if your bear yields any results for a particular
-check with a list of strings. First a *Section* and your Bear
-(in this case ``TooManyLinesBear``) is ``setUp``. Now your *Section* consists
-by default *Settings*. You can append any *Setting* depending on your test.
-Validate a check by passing your bear, lines to check as parameters
-(pass a few other parameters if necessary) to ``check_validity``. The method
-``self.check_validity(self.uut, ["import os"])`` asserts if your bear
-``self.uut`` yields a result when a list of strings ``["import os"]`` is
-passed.
+``check_validity`` asserts if your bear either yields or does not yield any
```
Comment (RaiVaibhav) [inline docs/Developers/Testing_Bears.rst]: >asserts if your bear either yields or does not yield

are you sure about this? if it does both then we can simply use it everywhere :P
I will recommend it to try `check_validity` at least once locally, things get clear :smiley: .
Also, this line is contradicting with line [103](https://github.com/coala/coala/pull/5483/files#diff-10a66d8d7f204ab29daecaf139504eaaR102)

Comment (RaiVaibhav) [reply docs/Developers/Testing_Bears.rst]: I will suggest you make the line 96 to `asserts if your bear does not yield any`

**Comment by stobis** [comment]
@jayvdb As you wish :)

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5483/commits/db9422a76f3ce601b927f4c315e5f1fa8e096ebc

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**